### DISCUSSION TITLE
Улучшение тест-кейса редактирования статьи

### DISCUSSION THREAD

**Discussion by Vivioza5** [open-tutorials/cypress#25] [Ideas]
Прохожу туториал [**фейковые данные**](https://github.com/open-tutorials/cypress/blob/8e0aafcf39e221f0e98954d7208dbf29f37bfbd0/tutorials/fake_data.md), выполняю в части написания кода для редактирования статьи.
 В части добавления, все норм, но при редактировании статьи как мне кажеться надо добавить пункт:

-  Clear all fields at article form.

Чтобы было более понятно после чего заполняем статью новыми данными и для чего мы добавляем в код функцию 

-  `clearArticle()` — очищает форму редактирования статьи.

Оно в принципе понятно, что если новые данные то надо очистить поля, но ведь по логике, если не указано очистить, то данные могут заполниться следом за предыдущими.

**Comment by Vladimir-Tarasov** [comment]
Думаю это правильное замечание. Спасибо. 
@breslavsky обрати внимание, пожалуйста. Думаю нам надо добавить в тест-кейс данный шаг.

**Reply by breslavsky** [reply]
@Vladimir4932 шли пул регвест.

**Reply by Vladimir-Tarasov** [reply]
ok

**Reply by Vladimir-Tarasov** [reply]
https://github.com/open-tutorials/cypress/pull/29
